tests/bulkdata/bulk_data_delete_tests.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.models import User
+from arches.app.models.resource import Resource
+from arches.app.models.tile import Tile
+from arches.app.models import models
+from arches.app.utils.data_management.resource_graphs.importer import import_graph as ResourceGraphImporter
+from arches.app.utils.betterJSONSerializer import JSONSerializer, JSONDeserializer
+from yourapp.models import BulkDataDeletion  # Adjust import as necessary
+from tests.arches_test_case import ArchesTestCase
+import os
+import uuid
+# these tests can be run from the command line via
+# python manage.py test tests.bulkdata.bulk_data_delete_tests --settings="tests.test_settings"
+
+class BulkDataDeletionTests(ArchesTestCase):
+    def setUpClass(self):
+        super().setUpClass()
+        # Setup test data here. You might want to create users, resources, tiles, etc.
+        self.user = User.objects.create(username='testuser', password='securepassword')
+        self.bulk_deleter = BulkDataDeletion()
+
+        with open(os.path.join("tests/fixtures/resource_graphs/Search Test Model.json"), "r") as f:
+            archesfile = JSONDeserializer().deserialize(f)
```

Comment:
Did you run this test locally in isolation?

This failed for me without:
```py
LanguageSynchronizer.synchronize_settings_with_db()
```

Reply:
fix in latest